frontend/package.json

```diff
@@ -59,6 +61,7 @@
     "redux-saga": "^1.1.3",
     "rimraf": "^3.0.2",
     "semver": "^7.3.4",
+    "styled-components": "^5.3.0",
```

Comment:
I'm not sure if we should add styled-components. Since we're already using the jss way everywhere else.